src/api.js

```diff
@@ -0,0 +1,8 @@
+export default {
+    HOST: 'https://api.giphy.com',
+    KEY: 'DJ3xIqeGfmTI3h1iNMK9uDeyLnAZWmxD',
```

Comment:
In real projects, avoid adding keys or secrets to the code repository. They are more secured in non-tracked env files.